lib/fizzbuzz.rb

```diff
@@ -3,7 +3,24 @@ class SuperFizzBuzz
   def run(input)
 
     #Implement your code here
+    return case input
```

Comment:
is the `return` doing anything for you here?
Pretty slick with the lambdas in any case.